rpc/nodeagent/storage/virtualharddisk/moc_nodeagent_virtualharddisk.proto

```diff
@@ -69,6 +69,7 @@ message VirtualHardDisk {
 	Entity entity = 18;
 	Tags tags = 19;
 	ImageSource sourceType = 20;
+	CloudInitDataSource cloudInitDataSource = 22;
```

Comment:
Is this to copy over the properties from GalleryImage to VHD on vmimage creation